### PR TITLE
[11.0][ADD] account_move_line_manufacture_info

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Add module account_move_line_manufacture_info to v11.

This module adds the manufacture information to any account move lines generated by a
manufacture operation such as a manufacturing order or an unbuild order.

### CODE REVIEW

File: account_move_line_manufacture_info/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    @api.model
+    def _prepare_account_move_line(self, qty, cost,
```
Comment (rousseldenis) [inline account_move_line_manufacture_info/models/stock_move.py:11]: @ageficent Maybe a little test on that ?

Comment (AdriaGForgeFlow) [reply account_move_line_manufacture_info/models/stock_move.py:11]: Done :+1:

**Review by AaronHForgeFlow — COMMENTED** [review]
:+1: Tested on runbot. MO correctly added to journal items.

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by AaronHForgeFlow — APPROVED** [review]

File: account_move_line_manufacture_info/views/mrp_production_views.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+<odoo>
+    <data>
+
+        <record id="mrp_production_form_view" model="ir.ui.view">
+            <field name="name">mrp.production.form</field>
+            <field name="model">mrp.production</field>
+            <field name="inherit_id" ref="mrp.mrp_production_form_view"/>
+            <field name="arch" type="xml">
+                <xpath expr="//div[hasclass('oe_button_box')]" position="inside">
+                    <button name="%(action_mrp_account_moves)d" type="action" string="Account Moves"
```
Comment (LoisRForgeFlow) [inline account_move_line_manufacture_info/views/mrp_production_views.xml:11]: Can you make these buttons only available to accounting users in a new PR?

Comment (LoisRForgeFlow) [reply account_move_line_manufacture_info/views/mrp_production_views.xml:11]: @ageficent

**Pull request merged**